migrations/00021_rewards.sql

```diff
@@ -2,13 +2,13 @@
 -- +goose StatementBegin
 CREATE TABLE rewards (
     issuance_week int NOT NULL,
+    vehicle_id int CONSTRAINT rewards_vehicle_token_id REFERENCES vehicles (id),
     connection_streak int,
-    streak_earnings int,
-    vehicle_id int NOT NULL CONSTRAINT rewards_vehicle_token_id REFERENCES vehicles (id),
+    streak_earnings numeric(30,0),
```

Comment:
Could you explain where 30 comes from in a comment? I do think it's wide enough, but I'd like to know why.

Reply:
You gave me an example in an earlier comment which was large, about 20. Just gave it a little more allowance in case of something higher.

Reply:
1 ether = 10^18 wei. The total supply, at the moment, is a billion. So 10^27, with 28 digits, is the biggest number you could possibly get.

Reply:
Could we add that comment?